provider/pkg/schema/gen.go

```diff
@@ -132,104 +135,9 @@ func GatherPackage(supportedResourceTypes []string, jsonSchemas []jsschema.Schem
 		Provider: pschema.ResourceSpec{
 			ObjectTypeSpec: pschema.ObjectTypeSpec{
 				Description: "The provider type for the AWS native package. By default, resources use package-wide configuration settings, however an explicit `Provider` instance may be created and passed during resource construction to achieve fine-grained programmatic control over provider settings. See the [documentation](https://www.pulumi.com/docs/reference/programming-model/#providers) for more information.",
-				Properties: map[string]pschema.PropertySpec{
```

Comment:
I believe the Properties defined here are redundant given the input properties below.

Reply:
`Properties` are outputs of the provider resource. Don't we define those in other provides?

Reply:
Doesn't seem like it? See https://github.com/pulumi/pulumi-kubernetes/blob/master/provider/pkg/gen/schema.go#L95 and https://github.com/pulumi/pulumi-kubernetes/blob/master/provider/pkg/gen/schema.go#L95.

Reply:
AWS does have them:
<img width="1664" alt="image" src="https://user-images.githubusercontent.com/1454008/142942938-96e5c2b8-e4a4-4786-a33a-acd8383ebe3c.png">

Reply:
I removed the properties change for now. I did mark the 3 sensitive fields as secrets though.